app/src/main/java/atlant/moviesapp/activity/TvShowDetails.java

```diff
@@ -78,29 +85,81 @@
     private Integer seasonNum;
     private Date date;
 
+    private BodyFavourite bodyFavourite;
+    private BodyWatchlist bodyWatchlist;
+
     @BindView(R.id.rate_txtBtn)
     TextView rateTxt;
     private static final int TAG = 1;
     @OnClick(R.id.rate_txtBtn)
     void rate()
     {
         Intent i = new Intent(this, RatingActivity.class);
-        i.putExtra("title","Rate this TV show");
-        i.putExtra("id",seriesId);
-        i.putExtra("tag",TAG);
+        i.putExtra(getString(R.string.title),getString(R.string.ratethisTVshow));
+        i.putExtra(getString(R.string.id),seriesId);
+        i.putExtra(getString(R.string.tag),TAG);
         startActivity(i);
 
     }
     @BindView(R.id.divider)
     TextView divider;
 
+    @OnClick(R.id.heart_detail)
```

Comment:
Please rename the method as it does the removing and adding. Example: updateFavourites()